### DISCUSSION TITLE
The filename of push to the repository was changed?!

### DISCUSSION THREAD

**Discussion by JackyChang-FoyaTech** [community/community#33105] [Other Feature Feedback, Questions, & Ideas]
Hi, I push a local commit to Github repository before 7 days ago. But, today one of header file be renamed. How possible this happen?
Has anyone met?
Thanks.

**Comment by jfullstackdev** [comment]
if the repo is public, maybe I can take a look

**Reply by JackyChang-FoyaTech** [reply]
It's private. Thank you.

**Reply by strong-728** [reply]
Hello, If the repo is private, did you invite anyone to that repo?
If so, the other guy may be changed that file name, not you.
In the case of the extreme, your github account info was shared to anyone.
Thanks.

**Reply by JackyChang-FoyaTech** [reply]
The file is created by me and push to github before 7 days ago. And no other one to change it in commit log.

**Reply by strong-728** [reply]
very strange. Please check this.
https://docs.github.com/en/repositories/working-with-files/managing-files/renaming-a-file
